ApiConnectorTests/TestHTTP.swift

```diff
@@ -0,0 +1,45 @@
+//
+//  TestHTTP.swift
+//  ApiConnector
+//
+//  Created by Mirellys on 27/03/2017.
+//  Copyright © 2017 WeAreReasonablePeople. All rights reserved.
+//
+
+import XCTest
+import ApiConnector
+
+class TestHTTP: XCTestCase {
+    
+    func testHTTPHashable() {
+        XCTAssertEqual(HTTP.Header.Key.accept, "Accept")
+        XCTAssertNotEqual(HTTP.Header.Key.accept, "Accept-Language")
+        XCTAssertEqual(HTTP.Header.Key.accept.hashValue, HTTP.Header.Key("Accept").hashValue)
+    }
+    
+    func testHTTHeader() {
```

Comment:
To be honest, I think that this test is a bit useless, as it tests `Dictionary`, which we know is working. So I think that you can remove completely this method